### PR TITLE
Typos from VisualSharp Microsoft

### PR DESCRIPTION
**Pull request by Fourchaux**
(Part 1)
cf. https://github.com/Microsoft/visualfsharp/pull/2789/files

**Note 1:** 
Line represnts->representative.
_representative_ is rather far from _represnts_, right?
Put _represents_ instead.

**Note 2:**
Line moutn->moutn
Line moutns->moutns
Are not these 2 (non)-corrections wrong?

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -13945,6 +13974,7 @@ unbeleivable->unbelievable
 unbeliefable->unbelievable
 unbelivable->unbelievable
 unce->once
+uncehcked->unchecked
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: I would also add above it `uncehck->uncheck`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11475,7 +11501,7 @@ represnets->represents
 represnt->represent
 represntative->representative
 represnted->represented
-represnts->representative
+represnts->represents
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: :+1:

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -8116,7 +8135,7 @@ limite->limit
 limitiation->limitation
 limitiations->limitations
 limitied->limited
-limted->limited
+limitted->limited
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Why was `limted` erased??

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Fourchaux) [reply codespell_lib/data/dictionary.txt]: ok.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Fourchaux) [reply codespell_lib/data/dictionary.txt]: (I don't know...)
Line restored.

**Comment by larsoner** [comment]
Travis
```
sort: codespell_lib/data/dictionary.txt:13463: disorder: texual->textual
Dictionary not sorted. Sort with 'make sort-dictionary'
```

**Comment by larsoner** [comment]
> Are not these 2 (non)-corrections wrong?

Yes I think so, do you want to change it here?

**Comment by Fourchaux** [comment]
There are still problems that I can correct so that this pr is accepted?

**Comment by luzpaz** [comment]
LGTM

**Pull request merged**